### PR TITLE
feat: create web pages with "About us" and "Cards"

### PR DESCRIPTION
**Pull request by vBrusentsov**

### CODE REVIEW

**Comment by anna-vrana-moc** [comment]
Hey @vBrusentsov please improve your code and remove fixed width and height in css. You can use only max-width for the parent block to limit the width of content.

**Comment by vBrusentsov** [comment]
Thank you. I ended refactored code according to your advice

**Comment by anna-vrana-moc** [comment]
@vBrusentsov You have several unresolved issues. You can find them by scrolling to the top of this conversation or by clicking on the 'Files' tab. Look for the comment right under the line that needs fixing

**Review by anna-vrana-moc — CHANGES_REQUESTED** [review]

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                    <li>
+                        <a href="./index.html">About</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main__section">
+            <img src="./Img/image.svg" width="532px" height="356px">
+            <div class="about-main__content">
+                <h1>About me</h1>
```
[the remark this answers is not in the document]
Comment (vBrusentsov) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html:26]: What are you mean margins?

[the remark this answers is not in the document]
Comment (anna-vrana-moc) [reply homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html]: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                        <a href="./index.html">About</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main__section">
+            <img src="./Img/image.svg" width="532px" height="356px">
+            <div class="about-main__content">
+                <h1 class="about-me__header">About me</h1>
+                <div class="main__paragraph">
```
Comment (anna-vrana-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html:28]: You don't need any div here, all styles you can apply to the p

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                    <p class="main-content">
+                        Hi! My name is Vladyslav Brusentsov and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.<br>
+                        <br>
```
Comment (anna-vrana-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/index.html:31]: This old practice of using tags like 'br', 'i', 'b' should be avoided. Please remove all 'br' tags and split the content by the <p> tag to create paragraphs in the layout

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/games.html [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+                        <a href="./index.html">About</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main__section">
+            <h1 class="main-header">Study project "Games"</h1>
+            <div class="main-cards-container">
+                <div class="card">
+                    <div class="main-card-header">
```
Comment (anna-vrana-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/games.html:28]: If you have several simular elements - it's a list. So, it is a list of cards. Please update the structure

File: homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/games.html [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+                        <img src="./Img/card-image.svg" width="90px" height="90px">
+                        <div class="card-header">
+                            <h3>Overwatch 2</h3>
+                            <p>Big changes come to the Overwatch formula in this sequel...</p>
+                        </div>
+                    </div>
+                    <div class="card-actors">
+                        <div class="card-actor-genre">
```
Comment (anna-vrana-moc) [inline homeworks/vladyslav.brusentsov_vBrusentsov/GamesProject/games.html:36]: card-actor-genre, card-actor-platform, card-actor-publisher, card-actor-developer, card-actor-release_date - don't have any specific styles, so remove them all

**Comment by anna-vrana-moc** [comment]
@vBrusentsov please check unresolved conversations (comments) and update your homework

**Review by anna-vrana-moc — APPROVED** [review]

**Pull request merged**